bin/kafka-streams-application-reset.sh

```diff
@@ -18,4 +18,4 @@ if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
     export KAFKA_HEAP_OPTS="-Xmx512M"
 fi
 
-exec $(dirname $0)/kafka-run-class.sh org.apache.kafka.tools.StreamsResetter "$@"
+exec $(dirname $0)/kafka-run-class.sh kafka.tools.StreamsResetter "$@"
```

Comment:
As I understand it, the plan is to move the tool to the Streams module once KIP-4 is in place so not clear the package before this change is the right one though.